WTRSimlib/src/main/java/simulation/launch/SimStarter.java

```diff
@@ -0,0 +1,50 @@
+package simulation.launch;
+
+import java.io.File;
+
+/**
+ * Run SimInit as a thread to launch and intialize simulator.
+ * 
+ * <p>If using a custom unity project path, pass the path as a string
+ * ex: "new SimInit(stringPathName).start();""
+ */
+public class SimStarter extends Thread {
+    String unityVersion = "2019.4.2f1";
+    String unityProjectPath;
+    String unityDefaultScenePath = File.separator 
+        + "Assets" + File.separator + "DefaultScene.unity";
+
+    Runnable socketHandler;
+    Runnable launchUnity;
+    
+
+    /**
+     * Default Constructor.
+     */
+    public SimStarter() {
+        this.unityProjectPath = System.getenv("userprofile") + File.separator 
+            + "Documents" + File.separator + "WTRSim" + File.separator 
+            + "WTRSimUnity";
```

Comment:
hm u might be right that a 80 character limit is too short, but here's my advised method to line wrap this
this.unityProjectPath =
        System.getenv("userprofile") + File.separator +
        "Documents" + File.separator +
        "WTRSim" + File.separator +
        "WTRSimUnity";

Reply:
Will make this change.